docs/orchestration/docker-compose/README.md

```diff
@@ -27,7 +27,7 @@ docker-compose.exe pull
 docker-compose.exe up
 ```
 
-Each instance is now accessible on the host at ports 9001 through 9004, proceed to access the Web browser at http://127.0.0.1:9001/
+Distributed instances is now accessible on the host at ports 9000, proceed to access the Web browser at http://127.0.0.1:9000/. Here 4 minio server instances are reverse proxyed through Nginx load balancing.
```

Comment:
```suggestion
Distributed instances are now accessible on the host at ports 9000, proceed to access the Web browser at http://127.0.0.1:9000/. Here 4 MinIO server instances are reverse proxied through Nginx load balancing.
```

Reply:
Resolved,thank you for your review. @harshavardhana